forge-game/src/main/java/forge/game/card/CardFactoryUtil.java

```diff
@@ -3018,7 +3018,7 @@ public boolean isForetelling() {
                 @Override
                 public void resolve() {
                     final Game game = getHostCard().getGame();
-                    final Card c = game.getAction().exile(getHostCard(), this);
+                    final Card c = game.getAction().exile(getHostCard(), this, null);
```

Comment:
Suspend + Foretell should also be done with CardCollection so Ranar ChangesZoneAll can trigger for it

Reply:
Hm OK it should use the CardCollection one?

Reply:
that seems reasonable